CRM/Report/Form.php

```diff
@@ -1781,6 +1780,12 @@ public function getOperationPair($type = "string", $fieldName = NULL) {
           'nll' => ts('Is empty (Null)'),
           'nnll' => ts('Is not empty (Null)'),
         ];
+
+        if ($fieldName == 'owner_membership_id') {
```

Comment:
@yashodha  this is a bit field-specificy for somewhere so central - I wonder if we should override in the specific report & put the field specific handling there? (ie call parent::getOperationPair for most of the logic)

Reply:
agree with @eileenmcnaughton, Say if you want to override in MembershipDetail report then:

```php
class MembershipDetailReport extends Report{
...
public function getOperationPair($type = "string", $fieldName = NULL) {
 parent::getOperationPair($type, $fieldName);
  if (in_array($type, [CRM_Report_Form::OP_INT, CRM_Report_Form::OP_FLOAT]) && $fieldName == 'owner_membership_id') {
     $result['nll'] = ts('Primary members only');
     $result['nnll'] = ts('Non-primary members only');
  }
}
...
}
```